unstract/sdk1/tests/test_prompt_caching.py

```
@@ -0,0 +1,313 @@
+        }
+    )
+    assert result["enable_prompt_caching"] is False
+
+
+# ── cost accounting: cached calls price against the cost_model override ──────
+
+
+def test_cost_override_passed_to_completion_cost_on_cached_call(monkeypatch) -> None:
+    """On a cached call, completion_cost must receive model= so the cost_model
+    override is honored (matching the cost_per_token fallback path).
+    """
```

Ruff (0.16.1)

[warning] 1 blank line required between summary line and description

(D205)